x-pack/plugins/security_solution/public/resolver/store/data/reducer.test.ts

```diff
@@ -40,7 +41,9 @@ describe('Resolver Data Middleware', () => {
       // Generate a 'tree' using the Resolver generator code. This structure isn't the same as what the API returns.
       const baseTree = generateBaseTree();
       const tree = mockResolverTree({
-        events: baseTree.allEvents,
+        // Casting here because the generator returns the SafeResolverEvent type which isn't yet compatible with
```

Comment:
👍 Created this issue to handle the front end side of things: https://github.com/elastic/security-team/issues/239

Reply:
Oh nice, thanks Mike!